bin/mutatex

```diff
@@ -299,25 +298,63 @@ try:
 except:
     exit(1)
 
+#get the original PDB list
```

Comment:
I think this part doesn't really belong to the part of the code in which we do mutations - we can prepare it directly right after we finish the pdb split/check phase, e.g. from line 239, or alternatively at the end of repair, e.g. from line 288

Reply:
No i agree, i have moved it up to the end of the repair step, this however, means that we call "`get_foldx_sequence(pdb, multimers=args.multimers)`" double. The current placement was there to be efficient and not introduce more computational time. What do you think now?

Reply:
I see - good point!

I think that in a long-term perspective I'd rather have a clear structure rather than save 200ms (or something) of execution time, so moving it sounds sensible